.github/workflows/opendata-ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Open Data CI
+on:
+  pull_request:
+    branches:
+      - dev
+    paths:
+      - 'src/open-data/**/*.csv'
+      - 'src/open-data/**/*.json'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env: 
+      CI_COMMIT_MESSAGE: Update open data
+      CI_COMMIT_AUTHOR: Open Data CI
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        token: ${{ secrets.WORKFLOW_GIT_ACCESS_TOKEN }}
```

Comment:
Should this secret be specific to open data or any PR update scenario? We should probably name it accordingly. If it's for any PR updates/validation, would GIT_PR_ACCESS_TOKEN or GIT_CICD_ACCESS_TOKEN make sense? 

@arthurclares Do you recall seeing any wave 1 or 2 security concerns around stored secrets?

Reply:
This could be used for any PR scenario yes, Im in favor of GIT_CICD_ACCESS_TOKEN personally

Reply:
Is read access to email addresses required? I don't see that option for orgs.
![image](https://github.com/user-attachments/assets/8da6c0aa-d57a-45ba-a33f-9c63651c8438)

Reply:
Ah, I did not test with an org token. I updated to use the built in token and specify the required permissions within the workflow itself. Here's the test in the fork https://github.com/aromano2/finops-toolkit/pull/17/commits.